.github/workflows/pkgdown.yml

```
@@ -0,0 +1,77 @@
+name: renderpkgdown
+
+on:
+  push:
+    branches:
+      - main
+      - develop
+    tags:
+      - '*'
+  pull_request:
+  merge_group:
+  workflow_dispatch:
+
+jobs:
+  pkgdown:
+    runs-on: ubuntu-latest  
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+
+    container: 
+      image: pecan/depends:develop-R4.1
+
+    steps:
+    # Checkout source code
+    - uses: actions/checkout@v4
+
+    # Install pkgdown
+    - name: Install dependencies
+      run: |
+        Rscript -e 'repos <- c(getOption("repos"), sub(r"(\d{4}-\d{2}-\d{2})", "latest", getOption("repos")))' \
+                 -e 'remotes::install_version("pkgdown", ">= 2.0.7", dependencies = TRUE, upgrade = FALSE, repos = repos)'
```

This was a hack to work around the R 4.1 base image finding a very old version of {bookdown} and isn't needed for pkgdown (nor for bookdown now that we run on R 4.4 by default!)

```suggestion
        Rscript -e 'remotes::install_version("pkgdown", ">= 2.0.7", dependencies = TRUE, upgrade = FALSE)'
```

---

Also: what pkgdown feature do we need >= 2.0.7 for? Can this be simplified all the way to...?
```suggestion
        Rscript -e 'install.packages("pkgdown")'
```